wegodang/users/views.py

```diff
@@ -1,3 +1,54 @@
-from django.shortcuts import render
+import jwt
+import requests
+import json
 
-# Create your views here.
\ No newline at end of file
+from django.http            import JsonResponse
+from django.core.exceptions import ValidationError
+from django.views           import View
+from django.conf            import settings
+
+from users.models           import User
+from users.validator        import validate_email
+
+class KaKaoSignUpView(View):
+    def get(self, request):
+        access_token   = request.GET.get('access_token')
+        KAKAO_INFO_API = 'https://kapi.kakao.com/v2/user/me'
+        user_info      = requests.get(KAKAO_INFO_API, headers={'Authorization': f'Bearer {access_token}'}).json()
```

Comment:
1. 변수명을 최대한 줄이지 말고 사용하세요.
2. requests 모듈을 사용할 때는 timeout 인자를 꼭 넣어 주셔야 합니다. 그래야 외부 요청이 잘 못 되었을 때 대응할 수 있습니다.
3. 유저 정보를 사용하기 위해 json()으로 바로 정보를 가져 왔는데, 이렇게 하면 requests로 받은 응답 값을 전부 활용하지 못 합니다. 그래서 먼저 응답을 변수에 담고 그 뒤에 body 정보나 status 정보를 활요하면 좋습니다.
4. requests를 통해 항상 올바른 응답이 오는 것은 아닙니다. 카카오로 요청 후 발생 할 수 있는 예외 상황에 대해 예외처리를 진행해 주세요.(예 권한이 없는 토큰으로 요청 했을 때)

```suggestion
        response      = requests.get(KAKAO_INFO_API, headers={'Authorization': f'Bearer {access_token}'}, timeout=5)
        user_information = response.json()
```

Reply:
수정했습니다 !